src/ctap/hid/mod.rs

```diff
@@ -50,7 +51,7 @@ pub enum ProcessedPacket<'a> {
 }
 
 // An assembled CTAPHID command.
-#[derive(Clone, Debug, PartialEq, Eq)]
+#[derive(Clone, Debug, PartialEq, Eq, Arbitrary)]
```

Comment:
This include should be conditional on being used for fuzzing, or it might unnecessarily increase binary size.